admin-ui/plugins/fido/components/DynamicConfiguration.tsx

```diff
@@ -143,12 +155,11 @@ function DynamicConfiguration({ fidoConfiguration, handleSubmit }) {
             label={fidoConstants.LABELS.LOGGING_LEVEL}
             name={fidoConstants.FORM_FIELDS.LOGGING_LEVEL}
             value={formik.values.loggingLevel}
-            defaultValue={formik.values.loggingLevel}
             values={fidoConstants.LOGGING_LEVELS}
             formik={formik}
             lsize={4}
             rsize={8}
-            showError={formik.errors.loggingLevel && formik.touched.loggingLevel}
+            showError={!!(formik.errors.loggingLevel && formik.touched.loggingLevel)}
             errorMessage={formik.errors.loggingLevel}
           />
```

Comment:
Standardize boolean coercion for showError

Only loggingLevel uses explicit boolean casting. Align other showError usages for consistency and to avoid truthy pitfalls.